docs/make.bat

```diff
@@ -58,206 +58,206 @@ goto sphinx_ok
 set SPHINXBUILD=python -m sphinx.__init__
 %SPHINXBUILD% 2> nul
 if errorlevel 9009 (
-	echo.
-	echo.The 'sphinx-build' command was not found. Make sure you have Sphinx
-	echo.installed, then set the SPHINXBUILD environment variable to point
-	echo.to the full path of the 'sphinx-build' executable. Alternatively you
-	echo.may add the Sphinx directory to PATH.
-	echo.
-	echo.If you don't have Sphinx installed, grab it from
-	echo.http://sphinx-doc.org/
-	exit /b 1
+    echo.
+    echo.The 'sphinx-build' command was not found. Make sure you have Sphinx
+    echo.installed, then set the SPHINXBUILD environment variable to point
+    echo.to the full path of the 'sphinx-build' executable. Alternatively you

```

Comment:
Is there a way to separate out all of these unrelated changes? (if not, fine as well, but it's easier to review without the autoformatter's whining)

Reply:
This is the "tab remover" that pybind11 has had forever (just updated to pre-commit). But I agree, tabs are evil. You can turn off whitespace in the diff on GitHub, though.

Reply:
Oh, OK. I like tabs, though, when used correctly, but yes, then it's fine. And didn't know about whitespace changes, thanks!